### DISCUSSION TITLE
First Sponsor payout is stuck

### DISCUSSION THREAD

**Discussion by sarthakroy2002** [community/community#61541] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My first payout from GitHub Sponsor was supposed to be in Jul 22... But it didn't arrive. Is any fault from my side? Please help me in this matter

**Comment by sarthakroy2002** [comment]
![Screenshot_20230723-094621](https://github.com/community/community/assets/47187468/b4c8fa3d-30b3-4aa0-8cd1-49378b9a768d)
It looks like this in Payouts of Stripe

**Comment by sarthakroy2002** [comment]
Any followup?

**Comment by antn** [comment]
👋 Hi @sarthakroy2002! I'm from the GitHub Sponsors team. We're investigating that might be happening here and will provide an update soon!

**Comment by sarthakroy2002** [comment]
Thanks... Waiting for a response ;)

**Comment by sarthakroy2002** [comment]
@antn any followup?

**Comment by rainbow1016** [comment]
very good question!

**Comment by antn** [comment]
Hi @sarthakroy2002, sorry for the delay here!

After some investigation, It looks like your initial payout wasn't sent out because it didn't meet the minimum payout threshold set by Stripe's banking partners in your region.

I do see that a payout for this month has been created and should arrive in your account today (the 22nd), since it was enough to be over that threshold.

If you're still running into any issues, please let us know!

**Comment by sarthakroy2002** [comment]
Thanks got it🫡

**Comment by Katekile2935** [comment]
Guys, I have a problem. I have been waiting for my payout for the full 90 days. Today is 22 July, and I'm still waiting for my payout. It is still not reflecting in my bank account.

![Screenshot_20260721_171648.jpg](https://github.com/user-attachments/assets/2c6a2d8d-d62b-4dd6-ae13-c96bdfe0b902)
